src/App.js

```diff
@@ -38,6 +38,15 @@ export default class App extends Component {
       this.setState(state)
     }
   }
+  onConfigSubmit = () => {
+    var config
+    try {
+      config = JSON.parse(this.refs.config.value)
+    } catch (error) {
```

Comment:
Not that I believe this makes any sense, but `standard` throws on this line if you do not include `error`, even though it is ultimately unused:

```
standard: Use JavaScript Standard Style (https://github.com/feross/standard) 
  /Users/Tim/Repos/react-player/src/App.js:45:14: Parsing error: Unexpected token
```